base_classes/NXpump.nxdl.xml

```diff
@@ -21,13 +21,13 @@
 #
 # For further information, see http://www.nexusformat.org
 -->
-<definition xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" category="base" name="NXpump" extends="NXcomponent" type="group" xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd">
+<definition xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" category="base" type="group" name="NXpump" extends="NXcomponent" xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd">
     <doc>
-         Device to reduce an atmosphere to a controlled remaining pressure level.
+        Device to reduce an atmosphere (real or simulated) to a controlled pressure.
     </doc>
-    <field name="design">
+    <field name="design" type="NX_CHAR">
         <doc>
-             Principle type of the pump.
+            Principle type of the pump.
         </doc>
         <enumeration>
             <item value="membrane"/>
```

Comment:
consider adding "ion", "cryo", "diffusion" and "scroll" type pumps. should also make the enumeration open since there are probably many more types.

Reply:
An extra field for "base pressure" would also be useful.